freqtrade/freqai/data_kitchen.py

```diff
@@ -1060,3 +1068,35 @@ def get_unique_classes_from_labels(self, dataframe: DataFrame) -> None:
         if self.unique_classes:
             for label in self.unique_classes:
                 self.unique_class_list += list(self.unique_classes[label])
+
+    def save_backtesting_prediction(
```

Comment:
Is there a reason you chose to put this outside `save_data()`? It is preferable to do this inside the data_kitchen `save_data()` function, as it is better to add another function. 

You can use the `if not self.live:` flag inside `save_data()` to do ensure this only occurs during backtesting.

Reply:
The main reason is because this method is called in freqai_interface after ```pred_df, do_preds = self.predict(dataframe_backtest, dk)``` that only happens some lines below, after the model is saved or reloaded (in case of a pre-existent model). I can change the logic, but I don't know if it's will be better. What do you think?

Reply:
Another idea came to my mind: Do we need to save the model in backtesting? Or could it be something optional defined in the config file? The reason to save the model would be to reuse it, however, we are already saving the predictions. In most cases, I think saving the model will be unnecessary in backtesting. This will improve first run performance too, right? What do you think?

Reply:
I agree we could add a config param to not save the model if you feel inclined to add it to the PR :)

Reply:
Sure. I will add it.